packages/atlas/src/components/_crt/CrtBasicInfoWidget/CrtBasicInfoWidget.tsx

```diff
@@ -1,64 +1,45 @@
-import styled from '@emotion/styled'
+import { ReactElement, ReactNode, useRef } from 'react'
 
-import { JoyTokenIcon } from '@/components/JoyTokenIcon'
+import { FlexBox } from '@/components/FlexBox'
+import { Color } from '@/components/Text/Text.styles'
+import { Tooltip } from '@/components/Tooltip'
 import { WidgetTile } from '@/components/WidgetTile'
 import { DetailsContent } from '@/components/_nft/NftTile'
 import { useMediaMatch } from '@/hooks/useMediaMatch'
-import { sizes } from '@/styles'
 
-type CrtBasicInfoWidgetProps = {
+export type CrtBasicInfoWidgetProps = {
   name?: string
-  totalRevenue?: string
-  revenueShare?: number
-  creatorReward?: number
+  titleColor?: Color
+  details: {
+    caption: string
+    content: number | string | ReactElement | ReactElement[]
+    icon?: ReactNode
+    tooltipText?: string
+  }[]
 }
 
-export const CrtBasicInfoWidget = ({ creatorReward, revenueShare, totalRevenue, name }: CrtBasicInfoWidgetProps) => {
+export const CrtBasicInfoWidget = ({ name, details, titleColor }: CrtBasicInfoWidgetProps) => {
+  const titleRef = useRef<HTMLSpanElement | null>(null)
   const smMatch = useMediaMatch('sm')
-
   return (
     <WidgetTile
-      title={`$${name || 'ABC'}`}
+      title={<span ref={titleRef}>${name ?? 'ABC'}</span>}
```

Comment:
Somehow the tooltip doesn't to show for me. Why not just:
```suggestion
      title={<span title="Token name">${name ?? 'ABC'}</span>}
```

Reply:
It's just because I used ref and the component does not rerender after it's set. The `title` attribute imo is too slow, especially since all of our tooltips appear almost instantly.

Replace useRef for useState